internal/exec/vendor_model.go

```
@@ -12,6 +12,7 @@
 	"github.com/charmbracelet/bubbles/spinner"
 	tea "github.com/charmbracelet/bubbletea"
 	"github.com/charmbracelet/lipgloss"
+	"github.com/charmbracelet/log"
```

golangci-lint (1.62.2)

import "github.com/charmbracelet/log" imported without alias but must be with alias "log" according to config

(importas)